dsc_lib/Cargo.toml

```diff
@@ -16,10 +16,11 @@ chrono = "0.4"
 clap = { version = "4.5", features = ["derive"] }
 derive_builder ="0.20"
 indicatif = "0.17"
-jsonschema = { version = "0.29", default-features = false }
+jsonschema = { version = "0.30", default-features = false }
 linked-hash-map = "0.5"
 num-traits = "0.2"
 regex = "1.11"
+rt-format = "0.3"
```

Comment:
This package seems reasonable enough, but I'll point out that it's last release was 3 years ago, and it has this note:

> Is stable API a must-have? If so, you might consider the alternatives. This crate is still not at version 1.0, which means that the API is still not completely stable.

Reply:
Just something to consider, [dyn-fmt](https://crates.io/crates/dyn-fmt) appears more recently updated than the other two contenders (rt-format as above and the similarly named dynfmt last published 4 years ago).

Reply:
(I'm not saying it's better, I'm just looking into it.)

Reply:
I was looking at the variety of options listed in the rt-format repository.  Maybe will switch to dyn-fmt in the future.  Since this is experimental, we can leave it as-is for now.